clang/test/CodeGen/X86/avx-cast-builtins.c

```diff
@@ -1,101 +1,230 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 5
```

Comment:
I think it makes sense to leave this test as-is... it's serving as an end-to-end test that the given C construct is a no-op in assembly.  If the output changes, that's a bug somewhere.  (We do similar things for other intrinsics we expect to lower to a specific sequence.)